conftest.py

```diff
@@ -712,6 +746,29 @@ def setup_test_colo(
         # Check to make sure that limit_db_cpus made it into the colo settings
         return colo_model
 
+
+@pytest.fixture(scope="function")
+def global_dragon_teardown() -> t.Generator[t.Any, t.Any, t.Any]:
```

Comment:
Pedantic typing thing: It doesn't look like the `yield` key word is used here, so this is actually just a normal function!
```suggestion
def global_dragon_teardown() -> None:
```